<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1700</width>
    <height>665</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="paytimesheetTable">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>60</y>
      <width>1701</width>
      <height>605</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>Poppins SemiBold</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::DefaultContextMenu</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">/* Header Design */

QHeaderView::section {
	font-family: Poppins;
	font-weight: bold;
	font-size: 13px;
	color: white;
	background-color: &quot;#344273&quot;;
	border: none;
}

/* Style for the selected row */
QTableView::item:selected:active {
	 selection-background-color: &quot;#9DADDF&quot;;
}

/* Style for the alternate row */
QTableWidget {
	alternate-background-color: &quot;#DDDDDD&quot;;
}</string>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAsNeeded</enum>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QAbstractScrollArea::AdjustIgnored</enum>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <property name="tabKeyNavigation">
     <bool>false</bool>
    </property>
    <property name="showDropIndicator" stdset="0">
     <bool>false</bool>
    </property>
    <property name="dragDropOverwriteMode">
     <bool>false</bool>
    </property>
    <property name="alternatingRowColors">
     <bool>true</bool>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::SingleSelection</enum>
    </property>
    <property name="selectionBehavior">
     <enum>QAbstractItemView::SelectRows</enum>
    </property>
    <property name="textElideMode">
     <enum>Qt::ElideRight</enum>
    </property>
    <property name="showGrid">
     <bool>false</bool>
    </property>
    <property name="gridStyle">
     <enum>Qt::SolidLine</enum>
    </property>
    <property name="sortingEnabled">
     <bool>false</bool>
    </property>
    <property name="cornerButtonEnabled">
     <bool>false</bool>
    </property>
    <property name="columnCount">
     <number>24</number>
    </property>
    <attribute name="horizontalHeaderVisible">
     <bool>true</bool>
    </attribute>
    <attribute name="horizontalHeaderCascadingSectionResizes">
     <bool>true</bool>
    </attribute>
    <attribute name="horizontalHeaderMinimumSectionSize">
     <number>50</number>
    </attribute>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>150</number>
    </attribute>
    <attribute name="horizontalHeaderHighlightSections">
     <bool>true</bool>
    </attribute>
    <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderStretchLastSection">
     <bool>true</bool>
    </attribute>
    <attribute name="verticalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderCascadingSectionResizes">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderMinimumSectionSize">
     <number>23</number>
    </attribute>
    <attribute name="verticalHeaderDefaultSectionSize">
     <number>31</number>
    </attribute>
    <attribute name="verticalHeaderHighlightSections">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderShowSortIndicator" stdset="0">
     <bool>false</bool>
    </attribute>
    <column>
     <property name="text">
      <string>Emp Number</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Bio Num</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Employee Name</string>
     </property>
     <property name="font">
      <font>
       <strikeout>false</strikeout>
      </font>
     </property>
     <property name="foreground">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </property>
    </column>
    <column>
     <property name="text">
      <string>CostCenter</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>DaysWork</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>DaysPresent</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>RestDay</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Holiday</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>RestHoliday</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>OrdinaryDayNight</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>RestDayNight</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>HolidayNight</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>RestHolidayNight</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>OrdinaryDayOT</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>RestDayOT</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>HolidayOT</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>RestHolidayOT</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Late</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Undertime</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Absent</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>DatePosted</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Remarks</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>EmpCompany</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>LegalHoliday</string>
     </property>
    </column>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1711</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: white;</string>
    </property>
    <widget class="QFrame" name="searchBar">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>361</width>
       <height>41</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::StrongFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">#searchBar{
	border: 1px solid &quot;#344273&quot;;
	border-radius: 3px; 
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLineEdit" name="txtSearch">
      <property name="geometry">
       <rect>
        <x>43</x>
        <y>10</y>
        <width>301</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Poppins</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="focusPolicy">
       <enum>Qt::ClickFocus</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">#txtSearch{

}</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="frame">
       <bool>false</bool>
      </property>
      <property name="placeholderText">
       <string>Type to search...</string>
      </property>
     </widget>
     <widget class="QLabel" name="searchIcon">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>31</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">#searchIcon{
	background-color: white;
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../../MainFrame/Resources/Icons/search.svg</pixmap>
      </property>
      <property name="scaledContents">
       <bool>false</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="lblDateFrom">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>20</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Poppins</family>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Date From:</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblFrom">
     <property name="geometry">
      <rect>
       <x>720</x>
       <y>20</y>
       <width>181</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Poppins</family>
       <pointsize>12</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="lblTo">
     <property name="geometry">
      <rect>
       <x>990</x>
       <y>20</y>
       <width>181</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Poppins</family>
       <pointsize>12</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="lblDateTo">
     <property name="geometry">
      <rect>
       <x>900</x>
       <y>20</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Poppins</family>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Date To:</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
